{"uid":"20da2bc08fb49e9f","name":"failed signup Test","fullName":"tests.sign_in_test.TestClass#test_01_failed_signup","historyId":"d473bdd714f5dca4321099c6e995022c","time":{"start":1756199956809,"stop":1756199977279,"duration":20470},"description":"Test to check failed sign up with incorrect email","descriptionHtml":"<p>Test to check failed sign up with incorrect email</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558464a5202a <unknown>\n#1 0x5584644f1a70 <unknown>\n#2 0x558464543907 <unknown>\n#3 0x558464543b01 <unknown>\n#4 0x558464591d54 <unknown>\n#5 0x55846456940d <unknown>\n#6 0x55846458f14f <unknown>\n#7 0x5584645691b3 <unknown>\n#8 0x55846453559b <unknown>\n#9 0x558464536971 <unknown>\n#10 0x558464a171fb <unknown>\n#11 0x558464a1af49 <unknown>\n#12 0x5584649fe2d9 <unknown>\n#13 0x558464a1baf8 <unknown>\n#14 0x5584649e2bbf <unknown>\n#15 0x558464a3f0b8 <unknown>\n#16 0x558464a3f296 <unknown>\n#17 0x558464a51006 <unknown>\n#18 0x7f92aea9caa4 <unknown>\n#19 0x7f92aeb29c3c <unknown>","statusTrace":"self = <tests.sign_in_test.TestClass object at 0x7f7b4a25da90>\n\n    @allure.severity(Severity.CRITICAL)\n    @allure.story(\"Sign Up Fail\")\n    @allure.description(\"Test to check failed sign up with incorrect email\")\n    @allure.title(\"failed signup Test\")\n    def test_01_failed_signup(self):\n        with allure.step(\"Accept cookies if present\"):\n            CookiePage(self.main_page.driver).accept_if_present()\n        with allure.step(\"Attempt to sign up with incorrect email\"):\n>           result = self.signin_page.incorrect_email(\"loyekab865@jxbav.com\")\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/sign_in_test.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/signin_page.py:64: in incorrect_email\n    self.wait_for_element(self.main_page.SIGN_IN, condition=\"clickable\")\npages/signin_page.py:51: in wait_for_element\n    return wait.until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"1aebc05a5ae568511fc7a12639bff722\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f7b4bf73880>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Wait until the method returns a value that is not False.\n    \n        Calls the method provided with the driver as an argument until the\n        return value does not evaluate to ``False``.\n    \n        Parameters:\n        -----------\n        method: callable(WebDriver)\n            - A callable object that takes a WebDriver instance as an argument.\n    \n        message: str\n            - Optional message for :exc:`TimeoutException`\n    \n        Return:\n        -------\n        object: T\n            - The result of the last call to `method`\n    \n        Raises:\n        -------\n        TimeoutException\n            - If 'method' does not return a truthy value within the WebDriverWait\n            object's timeout\n    \n        Example:\n        --------\n        >>> from selenium.webdriver.common.by import By\n        >>> from selenium.webdriver.support.ui import WebDriverWait\n        >>> from selenium.webdriver.support import expected_conditions as EC\n    \n        # Wait until an element is visible on the page\n        >>> wait = WebDriverWait(driver, 10)\n        >>> element = wait.until(EC.visibility_of_element_located((By.ID, \"exampleId\")))\n        >>> print(element.text)\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x558464a5202a <unknown>\nE       #1 0x5584644f1a70 <unknown>\nE       #2 0x558464543907 <unknown>\nE       #3 0x558464543b01 <unknown>\nE       #4 0x558464591d54 <unknown>\nE       #5 0x55846456940d <unknown>\nE       #6 0x55846458f14f <unknown>\nE       #7 0x5584645691b3 <unknown>\nE       #8 0x55846453559b <unknown>\nE       #9 0x558464536971 <unknown>\nE       #10 0x558464a171fb <unknown>\nE       #11 0x558464a1af49 <unknown>\nE       #12 0x5584649fe2d9 <unknown>\nE       #13 0x558464a1baf8 <unknown>\nE       #14 0x5584649e2bbf <unknown>\nE       #15 0x558464a3f0b8 <unknown>\nE       #16 0x558464a3f296 <unknown>\nE       #17 0x558464a51006 <unknown>\nE       #18 0x7f92aea9caa4 <unknown>\nE       #19 0x7f92aeb29c3c <unknown>\n\n.venv/lib/python3.13/site-packages/selenium/webdriver/support/wait.py:138: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"Test to check failed sign up with incorrect email","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558464a5202a <unknown>\n#1 0x5584644f1a70 <unknown>\n#2 0x558464543907 <unknown>\n#3 0x558464543b01 <unknown>\n#4 0x558464591d54 <unknown>\n#5 0x55846456940d <unknown>\n#6 0x55846458f14f <unknown>\n#7 0x5584645691b3 <unknown>\n#8 0x55846453559b <unknown>\n#9 0x558464536971 <unknown>\n#10 0x558464a171fb <unknown>\n#11 0x558464a1af49 <unknown>\n#12 0x5584649fe2d9 <unknown>\n#13 0x558464a1baf8 <unknown>\n#14 0x5584649e2bbf <unknown>\n#15 0x558464a3f0b8 <unknown>\n#16 0x558464a3f296 <unknown>\n#17 0x558464a51006 <unknown>\n#18 0x7f92aea9caa4 <unknown>\n#19 0x7f92aeb29c3c <unknown>","statusTrace":"self = <tests.sign_in_test.TestClass object at 0x7f7b4a25da90>\n\n    @allure.severity(Severity.CRITICAL)\n    @allure.story(\"Sign Up Fail\")\n    @allure.description(\"Test to check failed sign up with incorrect email\")\n    @allure.title(\"failed signup Test\")\n    def test_01_failed_signup(self):\n        with allure.step(\"Accept cookies if present\"):\n            CookiePage(self.main_page.driver).accept_if_present()\n        with allure.step(\"Attempt to sign up with incorrect email\"):\n>           result = self.signin_page.incorrect_email(\"loyekab865@jxbav.com\")\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/sign_in_test.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/signin_page.py:64: in incorrect_email\n    self.wait_for_element(self.main_page.SIGN_IN, condition=\"clickable\")\npages/signin_page.py:51: in wait_for_element\n    return wait.until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"1aebc05a5ae568511fc7a12639bff722\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f7b4bf73880>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Wait until the method returns a value that is not False.\n    \n        Calls the method provided with the driver as an argument until the\n        return value does not evaluate to ``False``.\n    \n        Parameters:\n        -----------\n        method: callable(WebDriver)\n            - A callable object that takes a WebDriver instance as an argument.\n    \n        message: str\n            - Optional message for :exc:`TimeoutException`\n    \n        Return:\n        -------\n        object: T\n            - The result of the last call to `method`\n    \n        Raises:\n        -------\n        TimeoutException\n            - If 'method' does not return a truthy value within the WebDriverWait\n            object's timeout\n    \n        Example:\n        --------\n        >>> from selenium.webdriver.common.by import By\n        >>> from selenium.webdriver.support.ui import WebDriverWait\n        >>> from selenium.webdriver.support import expected_conditions as EC\n    \n        # Wait until an element is visible on the page\n        >>> wait = WebDriverWait(driver, 10)\n        >>> element = wait.until(EC.visibility_of_element_located((By.ID, \"exampleId\")))\n        >>> print(element.text)\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x558464a5202a <unknown>\nE       #1 0x5584644f1a70 <unknown>\nE       #2 0x558464543907 <unknown>\nE       #3 0x558464543b01 <unknown>\nE       #4 0x558464591d54 <unknown>\nE       #5 0x55846456940d <unknown>\nE       #6 0x55846458f14f <unknown>\nE       #7 0x5584645691b3 <unknown>\nE       #8 0x55846453559b <unknown>\nE       #9 0x558464536971 <unknown>\nE       #10 0x558464a171fb <unknown>\nE       #11 0x558464a1af49 <unknown>\nE       #12 0x5584649fe2d9 <unknown>\nE       #13 0x558464a1baf8 <unknown>\nE       #14 0x5584649e2bbf <unknown>\nE       #15 0x558464a3f0b8 <unknown>\nE       #16 0x558464a3f296 <unknown>\nE       #17 0x558464a51006 <unknown>\nE       #18 0x7f92aea9caa4 <unknown>\nE       #19 0x7f92aeb29c3c <unknown>\n\n.venv/lib/python3.13/site-packages/selenium/webdriver/support/wait.py:138: TimeoutException","steps":[{"name":"Accept cookies if present","time":{"start":1756199956809,"stop":1756199967048,"duration":10239},"status":"passed","steps":[{"name":"Accept cookies if banner is present","time":{"start":1756199956809,"stop":1756199967048,"duration":10239},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":1,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Attempt to sign up with incorrect email","time":{"start":1756199967048,"stop":1756199977278,"duration":10230},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558464a5202a <unknown>\n#1 0x5584644f1a70 <unknown>\n#2 0x558464543907 <unknown>\n#3 0x558464543b01 <unknown>\n#4 0x558464591d54 <unknown>\n#5 0x55846456940d <unknown>\n#6 0x55846458f14f <unknown>\n#7 0x5584645691b3 <unknown>\n#8 0x55846453559b <unknown>\n#9 0x558464536971 <unknown>\n#10 0x558464a171fb <unknown>\n#11 0x558464a1af49 <unknown>\n#12 0x5584649fe2d9 <unknown>\n#13 0x558464a1baf8 <unknown>\n#14 0x5584649e2bbf <unknown>\n#15 0x558464a3f0b8 <unknown>\n#16 0x558464a3f296 <unknown>\n#17 0x558464a51006 <unknown>\n#18 0x7f92aea9caa4 <unknown>\n#19 0x7f92aeb29c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/shichor-python/shichor-python/tests/sign_in_test.py\", line 22, in test_01_failed_signup\n    result = self.signin_page.incorrect_email(\"loyekab865@jxbav.com\")\n  File \"/home/runner/work/shichor-python/shichor-python/.venv/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/shichor-python/shichor-python/pages/signin_page.py\", line 64, in incorrect_email\n    self.wait_for_element(self.main_page.SIGN_IN, condition=\"clickable\")\n    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/shichor-python/shichor-python/pages/signin_page.py\", line 51, in wait_for_element\n    return wait.until(EC.element_to_be_clickable(locator))\n           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/shichor-python/shichor-python/.venv/lib/python3.13/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Sign up with incorrect email: 'loyekab865@jxbav.com'","time":{"start":1756199967048,"stop":1756199977277,"duration":10229},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558464a5202a <unknown>\n#1 0x5584644f1a70 <unknown>\n#2 0x558464543907 <unknown>\n#3 0x558464543b01 <unknown>\n#4 0x558464591d54 <unknown>\n#5 0x55846456940d <unknown>\n#6 0x55846458f14f <unknown>\n#7 0x5584645691b3 <unknown>\n#8 0x55846453559b <unknown>\n#9 0x558464536971 <unknown>\n#10 0x558464a171fb <unknown>\n#11 0x558464a1af49 <unknown>\n#12 0x5584649fe2d9 <unknown>\n#13 0x558464a1baf8 <unknown>\n#14 0x5584649e2bbf <unknown>\n#15 0x558464a3f0b8 <unknown>\n#16 0x558464a3f296 <unknown>\n#17 0x558464a51006 <unknown>\n#18 0x7f92aea9caa4 <unknown>\n#19 0x7f92aeb29c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/shichor-python/shichor-python/.venv/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/shichor-python/shichor-python/pages/signin_page.py\", line 64, in incorrect_email\n    self.wait_for_element(self.main_page.SIGN_IN, condition=\"clickable\")\n    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/shichor-python/shichor-python/pages/signin_page.py\", line 51, in wait_for_element\n    return wait.until(EC.element_to_be_clickable(locator))\n           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/shichor-python/shichor-python/.venv/lib/python3.13/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"email","value":"'loyekab865@jxbav.com'"}],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":1,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"1375f7aac53aa8a5","name":"screenshot-call","source":"1375f7aac53aa8a5.png","type":"image/png","size":75809},{"uid":"adad50f88fd32e64","name":"page_source-call","source":"adad50f88fd32e64.html","type":"text/html","size":4220},{"uid":"7d0e29aa9e56a5d5","name":"console-call","source":"7d0e29aa9e56a5d5.txt","type":"text/plain","size":458}],"parameters":[],"stepsCount":4,"shouldDisplayMessage":true,"attachmentsCount":3,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Sign In Tests"},{"name":"severity","value":"critical"},{"name":"feature","value":"Sign In and Sign Up"},{"name":"story","value":"Sign Up Fail"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"sign_in_test"},{"name":"subSuite","value":"TestClass"},{"name":"host","value":"pkrvmqc4gcfdwos"},{"name":"thread","value":"2472-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.sign_in_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"20da2bc08fb49e9f.json","parameterValues":[]}